src/component/handlers/edit/editOnInput.js

```diff
@@ -40,7 +40,7 @@ function editOnInput(): void {
   var domSelection = global.getSelection();
 
   var {anchorNode, isCollapsed} = domSelection;
-  if (anchorNode.nodeType !== Node.TEXT_NODE) {
+  if (anchorNode.nodeType !== Node.TEXT_NODE && anchorNode.nodeType !== Node.ELEMENT_NODE) {
```

Comment:
What's the scenario here? For Android text replacement, the `span` can be the anchor node?